masterlist.yaml

```
@@ -8197,6 +8197,27 @@ plugins:
         util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 7
 
+  - name: 'Mentioned and Unused NPCs.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/51869/'
+        name: 'Mentioned and Unused NPCS - A Kvatchcount Classic'
+    req: [ 'DLCShiveringIsles.esp' ]
+    tag:
+      - Actors.ACBS
+      - Actors.AIData
+      - Actors.AIPackages
+      - Actors.Spells
+      - Factions
+      - Invent.Add
+      - Names
+      - Scripts
+    dirty:
+      - <<: *quickClean
+        crc: 0x06BB5AC6
+        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 25
+        udr: 17
```

The mod was updated (v1.1) & cleaned earlier today:

```suggestion
    clean:
      - crc: 0x7A06C373
        util: 'TES4Edit v4.0.4b'
```